Dionmok/Crowd-Sourced-Travel-Planner
# Fix: Update package-lock.json for ARM64 compatibility

## Changes
- Regenerated package-lock.json to resolve the "@rollup/rollup-darwin-arm64" dependency issue on M1/M2 Macs
- No changes to actual dependencies or functionality

## Testing Needed
Please test this branch on your development environments to ensure it works across different platforms:
- Intel/AMD machines
- ARM64 (M1/M2) Macs
- Windows/Linux systems

## How to Test
1. Pull this branch
2. Delete your node_modules folder
3. Run `npm install`
4. Run `npm run dev`
5. Verify the development server starts without Rollup errors

## Why This Change
The original package-lock.json was causing Rollup errors on ARM64 Mac systems. This update ensures compatibility while maintaining the same dependency versions.
